pkg/migrate/cstor/pool.go

```diff
@@ -86,6 +92,13 @@ func (c *CSPCMigrator) Migrate(name, namespace string) error {
 	if err != nil {
 		return err
 	}
+	if c.CSPCName == "" {
+		c.CSPCName = name
```

Comment:
nit: Instead of assaining directly we can make use of function `c.SetCSPCName(name)`  then it will full fill the meaning of that function.